compile_data.py

```diff
@@ -87,9 +99,9 @@ def get_evals_json(src_data_dir):
     parser.add_argument('--callback', default="", nargs="?",
                         help='JSONP callback to enable cross-domain requests. '
                              'Default: none')
-    parser.add_argument('-s', '--source-dir', default="data", nargs="?",
```

Comment:
Previously FCE exported data as several separate files, one per college. Since it is a single file now, it makes more sense to use file path instead of a folder, with `type=argparse.FileType('r')`